packages/babel-helpers/src/helpers.js

```diff
@@ -620,6 +620,10 @@ helpers.interopRequireWildcard = helper("7.0.0-beta.0")`
       return obj;
     }
 
+    if (!(obj instanceof Object)) {
```

Comment:
I wish we can use [`_.isObject`](https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L11743) in the helpers. I copied from there
```js
typeof obj === "object" || typeof obj === "function"
```
because `(new WeakMap).set(new Function, 42)` is valid.

Reply:
Thanks for the advices. Inspiring from [`_.isObject`](https://github.com/lodash/lodash/blob/es/isObject.js). We should also add null check, because `typeof null === 'object'` and `(new WeakMap).set(null, { default: null })` throws error.